cocos/base/CCTouch.h

```diff
@@ -101,13 +103,17 @@ class CC_DLL Touch : public Ref
      * @param id A given id
      * @param x A given x coordinate.
      * @param y A given y coordinate.
+     * @param force Current force for 3d touch.
+     * @param maxForce Maxium possible force for 3d touch.
      */
-    void setTouchInfo(int id, float x, float y)
+    void setTouchInfo(int id, float x, float y, float force = 0.0f, float maxForce = 0.0f)
```

Comment:
Please, avoid using default arguments.
The "cocos2d c++ guidelines" explains why we should avoid it:
https://github.com/cocos2d/cocos2d-x/wiki/Cpp--Coding-Style#default-arguments